### PR TITLE
WEBP support doc

### PR DESCRIPTION
**Pull request by maxime-rainville**
# Parent issue
- https://github.com/silverstripe/silverstripe-assets/issues/486

### CODE REVIEW

File: docs/en/02_Developer_Guides/14_Files/06_Allowed_file_types.md [hunk 1]
```diff
@@ -90,3 +90,43 @@ SilverStripe\MimeValidator\MimeUploadValidator:
...
       - 'text/calendar'
 ```
+
+## Adding new image types {#add-image-format}
+
+Silverstripe CMS support JPEG, GIF, PNG and WebP images format out of the box. Silverstripe CMS can be configured to support other less common images format (e.g.: AVIF). For this to work, your version of PHP and of the [`intervention/image` library](https://intervention.io/) must support these alternative image format.
+
+For example, this snippet can be added to the configuration of older Silverstripe CMS projects to allow them to work with WebP images. 
+
+```yml
```
Comment (maxime-rainville) [inline docs/en/02_Developer_Guides/14_Files/06_Allowed_file_types.md:100]: Initially, I thought of providing an example of how to add AVIF, but that's so new that only people with a specially complied version of PHP8.1 can use it. I thought WebP would be a better example even if it's only going to be relevant for people using Silverstripe CMS 4.10 or earlier.

File: docs/en/04_Changelogs/4.11.0.md [hunk 2]
```diff
@@ -23,6 +24,33 @@ In accordance with our [PHP support policy](/Getting_Started/Server_Requirements
...
+---
+Name: myproject-assetsfiletypes
+After: '#assetsfiletypes'
+---
+SilverStripe\Assets\File:
+  allowed_extensions:
+    webp: false
+```
+
+Read [Allowed file types](/Developer_Guides/Files/Allowed_file_types) in the Silverstripe CMS documentation for more information on how to enable or disable new image file format.
```
Comment (maxime-rainville) [inline docs/en/04_Changelogs/4.11.0.md:52]:
```suggestion
Read [Allowed file types](/Developer_Guides/Files/Allowed_file_types) in the Silverstripe CMS documentation for more information on how to enable or disable new image file formats.
```

File: docs/en/02_Developer_Guides/14_Files/06_Allowed_file_types.md [hunk 3]
```diff
@@ -90,3 +90,43 @@ SilverStripe\MimeValidator\MimeUploadValidator:
       - 'text/plain'
       - 'text/calendar'
 ```
+
+## Adding new image types {#add-image-format}
+
+Silverstripe CMS support JPEG, GIF, PNG and WebP images format out of the box. Silverstripe CMS can be configured to support other less common images format (e.g.: AVIF). For this to work, your version of PHP and of the [`intervention/image` library](https://intervention.io/) must support these alternative image format.
```
Comment (GuySartorelli) [inline docs/en/02_Developer_Guides/14_Files/06_Allowed_file_types.md:96]:
```suggestion
Silverstripe CMS support JPEG, GIF, PNG and WebP image formats out of the box and can be configured to support other less common image formats (e.g.: AVIF). For this to work, your version of PHP and of the [`intervention/image` library](https://intervention.io/) must support these alternative image formats.
```

File: docs/en/02_Developer_Guides/14_Files/06_Allowed_file_types.md (hunk 1 above)
Comment (GuySartorelli) [reply docs/en/02_Developer_Guides/14_Files/06_Allowed_file_types.md:100]: Fair. Even docs about AVIF would be stable eventually, and probably wouldn't get updated to say "older projects" like this does.

File: docs/en/04_Changelogs/4.11.0.md [hunk 4]
```diff
@@ -23,6 +24,33 @@ In accordance with our [PHP support policy](/Getting_Started/Server_Requirements
...
+
+WebP is an alternative image format for displaying picture on websites. It provides generally better results in most use cases to JPEG and PNG. 
+
+Read [An image format for the Web](https://developers.google.com/speed/webp) for more information about WebP.
+
+WebP has wide – but not universal – support across web browsers. Internet Explorer is the main browser that does not support WebP at this stage.
+
+Read [Can I use WebP?](https://caniuse.com/webp) to see which browsers can render WebP images.
+
+Until now, Silverstripe CMS would default to blocking content authors from uploading WebP images. Given that Internet Explorer is soon to be end-of-life and that its market share are now under 1% according to most surveys, we decided the time had come to enable WebP by default in the CMS.
```
Comment (GuySartorelli) [inline docs/en/04_Changelogs/4.11.0.md:37]: Maybe worth specifically mentioning that Internet Explorer goes EOL in June?
https://blogs.windows.com/windowsexperience/2021/05/19/the-future-of-internet-explorer-on-windows-10-is-in-microsoft-edge/

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: docs/en/02_Developer_Guides/14_Files/06_Allowed_file_types.md [hunk 5]
```diff
@@ -90,3 +90,43 @@ SilverStripe\MimeValidator\MimeUploadValidator:
...
+For example, this snippet can be added to the configuration of older Silverstripe CMS projects to allow them to work with WebP images. 
+
+```yml
+---
+Name: myproject-assetsfiletypes
+After: '#assetsfiletypes'
+---
+SilverStripe\Assets\File:
+  file_types:
+    webp: 'WebP Image File Format'
```
Comment (GuySartorelli) [inline docs/en/02_Developer_Guides/14_Files/06_Allowed_file_types.md:107]:
```suggestion
    webp: 'WebP Image'
```
May as well be consistent with #476

File: docs/en/02_Developer_Guides/14_Files/06_Allowed_file_types.md (hunk 3 above)
Comment (maxime-rainville) [reply docs/en/02_Developer_Guides/14_Files/06_Allowed_file_types.md:96]:
```suggestion
Silverstripe CMS support JPEG, GIF, PNG and WebP image formats out of the box. Silverstripe CMS can be configured to support other less common image formats (e.g.: AVIF). For this to work, your version of PHP and of the [`intervention/image` library](https://intervention.io/) must support these alternative image formats.
```

Comment (GuySartorelli) [reply docs/en/02_Developer_Guides/14_Files/06_Allowed_file_types.md:96]: Starting two sentences in a row with "Silverstripe CMS" reads a bit weird to me but it's not enough of an issue for me to push it. I'm fine with your suggested change.

File: docs/en/04_Changelogs/4.11.0.md (hunk 4 above)
Comment (maxime-rainville) [reply docs/en/04_Changelogs/4.11.0.md:37]:
```suggestion
Until now, Silverstripe CMS would default to blocking content authors from uploading WebP images. Given that [Internet Explorer will be end-of-life in June 2022](https://blogs.windows.com/windowsexperience/2021/05/19/the-future-of-internet-explorer-on-windows-10-is-in-microsoft-edge/) and that its market share are now under 1% according to most surveys, we decided the time had come to enable WebP by default in the CMS.
```

**Review by GuySartorelli — APPROVED** [review]
LGTM

**Pull request merged**